packages/astro/src/types/public/integrations.ts

```diff
@@ -115,6 +115,16 @@ export interface AstroAdapter {
 	 * If the adapter is not able to handle certain configurations, Astro will throw an error.
 	 */
 	supportedAstroFeatures: AstroAdapterFeatureMap;
+	/**
+	 * Runtime configuration that will be available to Astro's client-side code.
+	 */
+	runtimeConfig?: {
```

Comment:
We are already inside a configuration, so calling this `Config` seems redundant.

Also, "runtime" and client-side code have different meanings to us, so I would look for a better name or a better description.